test/functional/http/get/delegates.js

```diff
@@ -409,4 +410,93 @@ describe('GET /delegates', function () {
 			});
 		});
 	});
+
+	describe('GET /{publicKey}/forged', function () {
+
+		var forgedEndpoint = new swaggerEndpoint('GET /delegates/{publicKey}/forged');
+
+		describe('publicKey', function () {
+
+			it('using known address should be ok', function () {
+				return forgedEndpoint.makeRequest({ publicKey: validDelegate.publicKey }, 200).then(function (res) {
```

Comment:
That scenario can be handled very gracefully. Just add `minLength` to parameter. So if someone provide empty public key, it should throw invalid public key instead of empty object.